javascript/chronometer.js

```diff
@@ -1,26 +1,59 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0
+    this.intervalId = ''
   }
   startClick(callback) {
-    // ... your code goes here
+    this.intervalId = setInterval(() => {
+      if (callback) {
+      callback()}
+      if (this.currentTime > 100) {
+        this.currentTime = 0
+      }
```

Comment:
above conditional is preventing the currentTime from going pass 100, you might need this for time higher than 100 seconds.